RocketChat/Rocket.Chat.ReactNative
# Chore: Properly type MessageActions

## Proposed changes


## Issue(s)	


## How to test or reproduce

- Long press a message
- Make sure the bottom sheet is working properly
- Try on threads as well

## Screenshots

## Types of changes




## Checklist



## Further comments
